data/json/furniture.json

```diff
@@ -2190,7 +2190,10 @@
             "sound_fail": "whack!",
             "furn_set": "f_vending_o",
             "items": [
-                { "item": "glass_shard", "count": [1, 3] }
+                { "item": "glass_shard", "count": [1, 3] },
+                { "item": "reinforced_glass_pane", "count": [0, 1] },
```

Comment:
I'd say if it's `glass breaking!` then there couldn't be any intact reinforced glass pane.

Reply:
Makes sense. Fixed.